raiden/tests/utils/smoketest.py

```diff
@@ -373,104 +316,91 @@ def setup_raiden(transport, matrix_server, print_step, contracts_version, testch
         )
         args["service_registry_contract_address"] = service_registry_contract_address
 
-    return {
-        "args": args,
-        "contract_addresses": contract_addresses,
-        "ethereum_nodes": testchain_setup["node_executors"],
-        "token": token,
-    }
+    return {"args": args, "contract_addresses": contract_addresses, "token": token}
 
 
 def run_smoketest(
     print_step: Callable,
-    append_report: Callable,
     args: Dict[str, Any],
     contract_addresses: List[Address],
     token: ContractProxy,
-    debug: bool,
-    ethereum_nodes: List[HTTPExecutor],
 ):
     print_step("Starting Raiden")
 
-    config = deepcopy(App.DEFAULT_CONFIG)
-    args["config"] = config
-    # Should use basic routing in the smoke test for now
-    # TODO: If we ever utilize a PFS in the smoke test we
-    #     need to use the deployed service registry, register the
-    #     PFS service there and then change this argument.
-    args["routing_mode"] = RoutingMode.BASIC
-    args["one_to_n_contract_address"] = None
-
-    raiden_stdout = StringIO()
-    maybe_redirect_stdout = contextlib.redirect_stdout(raiden_stdout)
-    if debug:
-        maybe_redirect_stdout = contextlib.nullcontext()
```

Comment:
This is not related to the test itself. I moved it out and grouped with the other context managers.